compactcert/builder.go

```diff
@@ -321,7 +321,7 @@ func (ccw *Worker) tryBuilding() {
 
 	for rnd, b := range ccw.builders {
 		firstValid := ccw.ledger.Latest() + 1
-		acceptableWeight := ledger.AcceptableCompactCertWeight(b.votersHdr, firstValid)
+		acceptableWeight := ledger.AcceptableCompactCertWeight(b.votersHdr, firstValid, nil)
```

Comment:
nit: pass in `logging.Base()` instead of `nil` in
```golang
acceptableWeight := ledger.AcceptableCompactCertWeight(b.votersHdr, firstValid, nil)
```
so that you could avoid the `logger == nil` check in 
```golang
if logger == nil {
 	logger = logging.Base()
 }
```